src/dashboard/style.scss

```diff
@@ -484,6 +519,16 @@
 			justify-content: center;
 		}
 	}
+
+	.components-form-toggle.is-checked .components-form-toggle__track {
+		background-color: var(--main-link-color);
+	}
+
+	.components-textarea-control__input:focus,
```

Comment:
You can rewrite this as `:is(.components-textarea-control__input, .components-text-control__input):focus`

The `:is` is pretty powerful https://developer.mozilla.org/en-US/docs/Web/CSS/:is in clarity and utility.

Reply:
And changed the css selector, as recommended.